pages/dkp/konvoy/2.2/Install/index.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: layout.pug
+navigationTitle: Install
+title: Install
+menuWeight: 80
+excerpt: Install a Konvoy cluster
+beta: false
+enterprise: false
+---
+
+When installing DKP for a project, line-of-business, or enterprise, the first step is to determine the infrastructure on which you want to deploy.
+
+For example, you can:
+
+- Install on a public cloud infrastructure, such as Amazon Web Services (AWS) or Azure.
+- Install on an internal network on-premises with a physical (bare metal) or virtual infrastructure.
+- Install on air-gapped Edge.
+
+<p class="message--important"><strong>IMPORTANT: </strong>The infrastructure you select then determines the specific requirements for a successful installation.</p>
+
+
+Konvoy is a tool for provisioning Kubernetes clusters with a suite of pre-selected [Cloud Native Computing Foundation (CNCF)][cncf] and community-contributed tools.
+By combining a native Kubernetes cluster as its foundation with a default set of cluster extensions,
+Konvoy provides a complete out-of-the-box solution for organizations that want to deploy production-ready
+Kubernetes.
+
+This Quick Start guide provides simplified instructions to get your Konvoy cluster up and running with minimal configuration requirements on an Amazon Web Services (AWS) public cloud instance.
+
+# Before you begin
+
+Before installing Konvoy, ensure you have the [following](../prerequisites) items.
+
+# Installing Konvoy
+
+1.  Install required packages. In most cases, you can install the required software using your preferred package manager. For example, on a macOS computer, you can use [Homebrew][brew] to install `kubectl` and the `aws` command-line utility by running the following command:
+
+    ```bash
+    brew install kubernetes-cli awscli
+    ```
+
+1.  Check the Kubernetes client version. Many important Kubernetes functions **do not work** if your client is outdated. You can verify that the version of `kubectl` you have installed is supported by running the following command:
+
+    ```bash
+    kubectl version --short=true
+    ```
+
+1.  To download Konvoy, see the [Download Konvoy](../download) topic for information. You will need to download and extract the Konvoy package tarball.
+
+1.  Install with default settings.
```

Comment:
Do we have/need a command here? Or is it self-explanatory? Not sure...

Reply:
I think it is just saying to accept the "defaults" at this point of installation, but I will defer to Dimitri to verify.